docs/components/logs-sharing.rst

```diff
@@ -1,7 +1,9 @@
 Logs Sharing
 ------------
 
-.. warning::
+:code:`--share-logs`
+""""""""""""""""""""
+.. info::
```

Comment:
Please add a line between line 5 and 6.

Reply:
I'll give you 4 `\n` for the price of one :smile: